ext/libxml/libxml.c

```diff
@@ -22,6 +22,7 @@
 #include "php.h"
 #include "SAPI.h"
 
+#include "zend_attributes.h"
```

Comment:
@kocsismate Should the stub generator generate the `#include "zend_attributes.h"` into the `arginfo.h`? It should not be necessary to explicitly include `zend_attributes.h` yourself.

Reply:
Good question... but currently it neither generates any other includes... Maybe it's a good idea, but I am not sure about the consequences fully. E.g. whether there is a situation when the generated include wouldn't work for some reason (and thus one has to reference the header file some other way).

Reply:
> but currently it neither generates any other includes...

Yes, that's the reason I asked first before sending a PR.

> E.g. whether there is a situation when the generated include wouldn't work for some reason (and thus one has to reference the header file some other way).

There could possibly be a few situations where the arginfo then implicitly includes a system header, without setting a `#define`. But I expect that to easily be resolved by just moving the `#include` of the `arginfo.h` to the bottom, including it last. Any header that is required for something else will already be included and all defines be set.

Reply:
Unfortunately, I don't feel confident enough to implement this change, so I'll leave the possibility for you :)